### ISSUE TITLE
Support excluding build info module properties

### ISSUE THREAD

**Issue by dg424** [jfrog/artifactory-maven-plugin#56]
### Is your feature request related to a problem? Please describe.

Yes, please see question posted here - https://github.com/jfrog/artifactory-maven-plugin/issues/55 - related to fix here - https://github.com/jfrog/jenkins-artifactory-plugin/pull/720

### Describe the solution you'd like to see

Please implement the fix from the Jenkins plugin into the maven plugin as well.

**Comment by rquinio1A** [comment]
+1, there should be a big warning that build-info captures all Maven properties, thus sensitive data will leak if passed as a Maven property.

**Comment by ddag** [comment]
> +1, there should be a big warning that build-info captures all Maven properties, thus sensitive data will leak if passed as a Maven property.

Yes, but why not just port same fox that's already in the Jenkins plug-in ? It's the same base code I believe, but the Jenkins code does not pass those along.
